content/ja/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm.md

```diff
@@ -509,10 +509,10 @@ and `scp` using that other user instead.
 
 The `admin.conf` file gives the user _superuser_ privileges over the cluster.
 This file should be used sparingly. For normal users, it's recommended to
-generate an unique credential to which you whitelist privileges. You can do
+generate an unique credential to which you allowlist privileges. You can do
```

Comment:
Here `allowlist` is a verb?

Reply:
I believe it's a noun.

Reply:
I've removed this whole file since it is a non-English localization file.